monai/transforms/post/dictionary.py

```diff
@@ -818,6 +822,56 @@ def __call__(self, data: Mapping[Hashable, NdarrayOrTensor]) -> Dict[Hashable, N
         return d
 
 
+class GetInstanceLevelSegMapd(MapTransform):
+    """Process Nuclei Prediction with XY Coordinate Map.
+
+    Args:
+        keys: keys of the corresponding items to model output.
+        hover_key: keys of the probability map from nucleus prediction branch.
+        threshold_pred: threshold the float values of prediction to int 0 or 1 with specified theashold. Defaults to 0.5.
+        threshold_overall: threshold the float values of overall gradient map to int 0 or 1 with specified theashold.
+            Defaults to 0.4.
+        min_size: objects smaller than this size are removed. Defaults to 10.
+        sigma: std. could be a single value, or `spatial_dims` number of values. Defaults to 0.4.
+        kernel_size: the size of the Sobel kernel. Defaults to 17.
+        radius: the radius of the disk-shaped footprint. Defaults to 2.
+        allow_missing_keys: don't raise exception if key is missing.
+
+    """
+
+    backend = GetInstanceLevelSegMap.backend
+
+    def __init__(
+        self,
+        keys: KeysCollection,
+        hover_key: str,
+        threshold_pred: float = 0.5,
+        threshold_overall: float = 0.4,
+        min_size: int = 10,
+        sigma: Union[Sequence[float], float, Sequence[torch.Tensor], torch.Tensor] = 0.4,
+        kernel_size: int = 17,
+        radius: int = 2,
+        allow_missing_keys: bool = False,
+    ) -> None:
+        super().__init__(keys, allow_missing_keys)
+        self.hover_key = hover_key
+        self.transform = GetInstanceLevelSegMap(
+            threshold_pred=threshold_pred,
+            threshold_overall=threshold_overall,
+            min_size=min_size,
+            sigma=sigma,
+            kernel_size=kernel_size,
+            radius=radius,
+        )
+
+    def __call__(self, data: Mapping[Hashable, NdarrayOrTensor]) -> Dict[Hashable, torch.Tensor]:
+        d = dict(data)
+        for key in self.key_iterator(d):
+            d[key] = self.transform(d[key], d[self.hover_key])
+
+        return d  # type: ignore
```

Comment:
May I ask why do you need to ignore type checking here?

Reply:
Did you try declaring d with a type hint? That might do it:

i.e. on line 868 'd: Dict[Hashable, torch.Tensor] = dict(data)`

Reply:
Hi @JHancox, thanks for your reply! I just fixed it in https://github.com/Project-MONAI/MONAI/pull/5173/commits/d024cfe72274ab8eee6adda3c98a67532eaa17da.